antismash/common/secmet/features/feature.py

```diff
@@ -51,13 +51,14 @@ class Feature:
         subclass.
     """
     __slots__ = ["location", "notes", "type", "_qualifiers", "created_by_antismash",
-                 "_original_codon_start"]
+                 "_original_codon_start", "_pseudo"]
 
     def __init__(self, location: FeatureLocation, feature_type: str,
                  created_by_antismash: bool = False) -> None:
         assert isinstance(location, (FeatureLocation, CompoundLocation)), type(location)
         if location_bridges_origin(location):
             raise ValueError("Features that bridge the record origin cannot be directly created: %s" % location)
+        self._pseudo = False # direct creation is only allowed with falsy value
```

Comment:
Comments should typically indicate _why_ the code does something, rather than repeating what the code says.

```suggestion
        self._pseudo = False # only ever set to True in from_biopython() when relevant
```